web-client/views/upload.jade

```diff
@@ -240,13 +240,16 @@ html
           br
           p(class='sideHeader')  &nbsp; Viewport Size
           input(type='radio' id='boundBoxS' class='boundBoxSize' name='size' value='containerS')
-          label(for='boundBoxS' style='font-weight:normal' class='sideLabel') Small (1104x648px)
+          label(for='boundBoxS' style='font-weight:normal' class='sideLabel') Small
+          small (1104x648px)
```

Comment:
Yes, it does make a difference!

![screen shot 2017-04-28 at 5 50 56 pm](https://cloud.githubusercontent.com/assets/6444156/25551550/432b43ee-2c3b-11e7-8f93-8f0e41e12341.png)